runtime/proto/queue.proto

```diff
@@ -16,13 +16,19 @@ message CorfuQueueMetadataMsg {
     int64 txSequence = 1;
 }
 
+enum ReplicationType {
+    LOG_ENTRY = 0; // Default value indicating absence of any full sync.
+    FULL_SYNC = 1; // This entry was made during a full sync
```

Comment:
let us use the convention of 'Log Entry Sync' and 'Snapshot Sync'

Reply:
done.